tests/console/validate_vnc_avail.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
```

Comment:
I didn't see this idea on the ticket, but it is true that it seems the only way to know for real that the installed system doesn't stop the connections for some reason, I would suggest to include it in different commit or to amend current one to reflect validation in both sides no only in target/server.

Reply:
I don't really understand your proposal here. Which parts would be better to be separated? Would that be something significant? To separate the commits would be chaotic for me. It would be easier to copy all the changes to a new branch.

Reply:
It is good you get used to it, to have separate commits, but sometimes it is a bit tricky, if simplify things for you, just reflect in the commit message that the validation are in both sides, not only in the target like now it looks: "Add validation for remote_vnc_target_nfs". wdyt?

Reply:
Ok